src/Middleware/HttpOverrides/src/ForwardedHeadersOptions.cs

```diff
@@ -82,8 +84,15 @@ public class ForwardedHeadersOptions
 
     /// <summary>
     /// Address ranges of known proxies to accept forwarded headers from.
+    /// Obsolete, please use <see cref="KnownIPNetworks"/> instead
     /// </summary>
-    public IList<IPNetwork> KnownNetworks { get; } = new List<IPNetwork>() { new IPNetwork(IPAddress.Loopback, 8) };
+    [Obsolete("Please use KnownIPNetworks instead")]
```

Comment:
We're trying to align some of our obsoletion messages, could we update these to:
`[Obsolete("Please use KnownIPNetworks instead. For more information, visit https://aka.ms/aspnet/deprecate/004.", DiagnosticId = "ASPDEPR004")]`

That also means we'll need to update the pragma warnings to use ASPDEPR004.

Reply:
Should we update the `HttpOverrider.IPNetwork` obsolete message also? Is there a special diagnosticId?

Reply:
Yeah, can use the same ID and url link